DataFormats/Detectors/TRD/test/testRawData.cxx

```diff
@@ -109,15 +109,15 @@ BOOST_AUTO_TEST_CASE(TRDRawDataHeaderInternals)
   tracklets[1].word = 0;
   tracklets[2].word = 0;
   std::array<uint8_t, 3> charges;
-  auto invalidheader = getChargesFromRawHeaders(hcheader, &header, tracklets, charges, 0);
-  auto trackletcount = getNumberOfTrackletsFromHeader(&header);
+  // auto invalidheader = getChargesFromRawHeaders(hcheader, &header, tracklets, charges, 0);
+  // auto trackletcount = getNumberOfTrackletsFromHeader(&header);
 
-  BOOST_CHECK(trackletcount == 1);
-  BOOST_CHECK(invalidheader == 0);
-  //std::cout << std::hex << " charges[0]:0x"<<(int)charges[0];
-  //std::cout << std::hex << " charges[1]:0x"<<(int)charges[1];
-  //std::cout << std::hex << " charges[2]:0x"<<(int)charges[2];
-  BOOST_CHECK((int)charges[2] == 0x02);
+  // BOOST_CHECK(trackletcount == 1);
+  // BOOST_CHECK(invalidheader == 0);
```

Comment:
Is there a reason why these checks were removed?

Reply:
Yes, it is because the functions they rely on were removed. And I removed them since they are not needed anymore by the new parsing. If one would want to test that here one would need to create the data for one link and then use `CruRawReader::parseTrackletLinkData()` and compare input/output.
As a test I used the chain from simulation to raw data and then again to digits and tracklets and made sure that that is consistent.